README.md

```diff
@@ -1,24 +1,112 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget-app
 
-Things you may want to cover:
+> You will create a Ruby on Rails application that allows the user to:
 
-* Ruby version
+> - register and log in, so that the data is private to them.
+> - introduce new transactions associated with a category.
+> - see the money spent on each category.
 
-* System dependencies
+## Built With
 
-* Configuration
+- Ruby on Rails
+- Boostrap
+- Postgres SQL
+- Rspec
+- Capybara
 
-* Database creation
+## Live Demo (if available)
 
-* Database initialization
+[live demo](https://morris-coin-app.herokuapp.com/)
 
-* How to run the test suite
+## Getting Started
 
-* Services (job queues, cache servers, search engines, etc.)
+To get a local copy up and running follow these simple example steps.
 
-* Deployment instructions
+### Prerequisites
 
-* ...
+- You should have ruby installed in your machine, you can follow the steps given by the [official documentation](https://www.ruby-lang.org/en/documentation/installation/).
+
+- You should have a postgres user with superuser permissions. you can refer to the official [postgres documentation](https://www.postgresql.org/docs/current/role-attributes.html#:~:text=To%20create%20a%20new%20database,that%20is%20already%20a%20superuser.&text=A%20role%20must%20be%20explicitly,use%20CREATE%20ROLE%20name%20CREATEDB%20.) to create or update a role.
+
+### Setup
+
+First, you must clone this repository locally running this command:
+
+```
+git clone git@github.com:marurevi/Budget-app.git
+```
+
+and navigate to the cloned directory:
+
+```
+cd budget_app
+```
+
+### Install
+
+Once inside the project directory, you should install all project dependencies:
+
+- Install bundler:
+
+```
+gem install bundler
+```
+
+- Install project dependencies
+
+```
+bundle install
+```
+
+### Usage
+
+You can get the app running locally by opening a local server:
+
+```
+bin/rails server
+```
+
+### Run tests
+
+If you're on a linux-based system, you could run the executable file to run all specs:
+
+```
+bin/rspec
+```
+
+Windows users have to specify ruby in this case:
+
+```
+ruby bin/rspec
+```
+
+## Authors
```

Comment:
- [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.